packages/shared/src/utils/config/config.js

```diff
@@ -79,28 +79,26 @@ export const getSocketURL = () => {
 };
 
 export const checkAndSetEndpointFromUrl = () => {
-    if (isStaging() || isTestLink()) {
-        const url_params = new URLSearchParams(location.search.slice(1));
+    const url_params = new URLSearchParams(location.search.slice(1));
 
-        if (url_params.has('qa_server') && url_params.has('app_id')) {
-            const qa_server = url_params.get('qa_server');
-            const app_id = url_params.get('app_id');
+    if (url_params.has('qa_server') && url_params.has('app_id')) {
+        const qa_server = url_params.get('qa_server');
+        const app_id = url_params.get('app_id');
 
-            url_params.delete('qa_server');
-            url_params.delete('app_id');
+        url_params.delete('qa_server');
+        url_params.delete('app_id');
 
-            if (/^(www\.binaryqa[0-9]{1,2}\.com|(.*)\.binaryws\.com)$/.test(qa_server) && /^[0-9]+$/.test(app_id)) {
-                localStorage.setItem('config.app_id', app_id);
-                localStorage.setItem('config.server_url', qa_server);
-            }
+        if (/^((www\.)?binaryqa[0-9]{1,2}\.com|(.*)\.binaryws\.com)$/.test(qa_server) && /^[0-9]+$/.test(app_id)) {
```

Comment:
there are no `binaryqaXX.com` domains anymore , can you change this to support `qaXX.deriv.dev`?

Reply:
thank you!, yup, missed that one